js/ui/action_button/action_button.js

```diff
@@ -0,0 +1,138 @@
+import registerComponent from "../../core/component_registrator";
+import { extend } from "../../core/utils/extend";
+import Guid from "../../core/guid";
+import Widget from "../widget/ui.widget";
+import themes from "../themes";
+import { init, dispose } from "./action_button_base";
+
+var FloatingActionButton = Widget.inherit({
+    _getDefaultOptions: function() {
+        return extend(this.callBase(), {
+            /**
+            * @name dxFloatingActionButtonOptions.icon
+            * @type string
+            * @default ""
+            */
+            icon: "",
+
+            /**
+            * @name dxFloatingActionButtonOptions.label
+            * @type string
+            * @default ""
+            */
+            label: "",
+
+            /**
+            * @name dxFloatingActionButtonOptions.onClick
+            * @type function(e)
+            * @type_function_param1 e:object
+            * @type_function_param1_field1 event:event
+            * @type_function_param1_field2 component:this
+            * @type_function_param1_field3 element:dxElement
+            * @action
+            */
+
+            onClick: null,
+
+            useInkRipple: false,
+
+            /**
+            * @name dxFloatingActionButtonOptions.visible
+            * @hidden
+            * @inheritdoc
+            */
+            visible: false,
+
+            /**
+            * @name dxFloatingActionButtonOptions.width
+            * @hidden
+            * @inheritdoc
+            */
+            width: "auto",
```

Comment:
Is any reason to declare `width` and `height` properties? They are undefined by default. It means that no `style` attribute will added. And width and height will be `auto`. With `visible:false` it has less sense.